app/config/FrontendAppConfig.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 HM Revenue & Customs
+ * Copyright 2023 HM Revenue & Customs
```

Comment:
After the unused config in `application.conf` is cleaned up, there are unused vals here that can be removed. Can you remove: `reportAProblemPartialUrl`, `reportAProblemNonJSUrl`, `betaFeedbackUrl`, `betaFeedbackUnauthenticatedUrl`, `renewalProgressUrl`, `registrationProgressUrl`, `authUrl`, `loginContinueUrl`.